src/main/java/com/example/EnumDemo/dto/showDressDto.java

```diff
@@ -0,0 +1,19 @@
+package com.example.EnumDemo.dto;
+
+import com.example.EnumDemo.enumC.Color;
+import com.example.EnumDemo.enumC.Size;
+import com.example.EnumDemo.enumC.Type;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class showDressDto {
```

Comment:
Name of the class should follow camelcase
This name sound more like a method

Reply:
Done